.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -18,12 +18,16 @@ jobs:
       run: |
         set -eu
         since_time=$(date -u -d '24 hours ago' '+%a, %d %b %Y %H:%M:%S GMT')
+        outfile=./code.txt
         curl \
             -o nvd.zip \
             -w "%{http_code}" \
             -H "If-Modified-Since: $since_time" \
             https://definitions.stackrox.io/v4/nvd/nvd.zip \
-        | grep 200
+        > $outfile
+
+        echo "code: $(cat $outfile)"
+        grep 200 $outfile
```

Comment:
```suggestion
        | while read code _; do
            echo >&2 "NVD bucket status code: $code"
            echo $code
        done | grep -q 200
        echo "Status code successful"
```

Reply:
Doesn't appear to work:

```sh
davidc-dev in ~ 
❯         curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            https://definitions.stackrox.io/v4/nvd/nvd.zip \
        | while read code; do
            echo >&2 "NVD bucket status code: $code"
            echo $code
        done | grep -q 200
        echo "Status code successful"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Status code successful

davidc-dev in ~ 
❯ curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            https://definitions.stackrox.io/v4/nvd/nvd.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
304
```

Reply:
Please, remove the `while`, see current suggestion.

Reply:
```sh
        curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            https://definitions.stackrox.io/v4/nvd/nvd.zip \
        | read code _
          echo "NVD bucket status code: $code"
          echo $code | grep -q 200
        echo "Status code successful"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
NVD bucket status code: 
Status code successful
```

Reply:
Wonder if this will work with no interactive terminal:
```
        curl \
            -o nvd.zip \
            -w "%{http_code}" \
            -H "If-Modified-Since: $since_time" \
            https://definitions.stackrox.io/v4/nvd/nvd.zip \
        | tee /dev/tty | grep -q 200
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
304
```